<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_rule</class>
 <widget class="QWidget" name="Add_rule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>55</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Source IP address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>105</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Source port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>155</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination IP address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>205</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="commit_button">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>360</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>commit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>255</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Protocol</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sip_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>55</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="sip_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>55</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="sip_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>55</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="sip_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>55</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="dstip_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>155</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="dstip_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>155</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="dstip_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>155</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="dstip_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>155</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="sport">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>105</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="dport">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>205</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="protocol">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>255</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>45</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="result">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>771</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Feedback:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>45</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>45</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>145</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>145</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>145</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>85</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>275</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(ICMP/UDP/TCP)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>55</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>105</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>End time</string>
   </property>
  </widget>
  <widget class="QComboBox" name="date">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>27</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>29</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>31</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Week</string>
   </property>
  </widget>
  <widget class="QComboBox" name="week">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sunday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tuesday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wednesday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thursday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Friday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Saturday</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="start_time">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>55</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_time">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>105</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(00:00-24:00)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(00:00-24:00)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>165</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Indev mac</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Outdev mac</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="indev_mac">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="outdev_mac">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>210</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ICMP Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ICMP_type">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="icmp_help">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>260</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>view help</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
